packages/ts/src/components/nested-donut/style.ts

```diff
@@ -0,0 +1,82 @@
+import { css } from '@emotion/css'
+
+// Utils
+import { getCssVarNames, injectGlobalCssVariables } from 'utils/style'
+
+export const labelColors = {
+  dark: '#5b5f6d',
+  light: '#fff',
+}
+
+const cssVarDefaults = {
+  '--vis-nested-donut-background-color': '#E7E9F3',
+  // Undefined by default to allow proper fallback to var(--vis-font-family)
+  '--vis-nested-donut-font-family': undefined,
+
+  // Central label
+  '--vis-nested-donut-central-label-font-size': '16px',
+  '--vis-nested-donut-central-label-font-weight': 600,
+  '--vis-nested-donut-central-label-text-color': labelColors.dark,
```

Comment:
I would also add CSS variables for dark / light segment labels, like we have it in Graph or LeafletMap:
```
'--vis-map-point-inner-label-text-color-dark': '#5b5f6d',
'--vis-map-point-inner-label-text-color-light': '#fff',
```

it looks like now there's no way to change default `labelColors` for segments.